### PR TITLE
modern-uit-thesis:0.1.0

### PR DESCRIPTION
**Pull request by otytlandsvik**
I am submitting
- [x] a new package
- [ ] an update for a package

This is a template intended for students to write their bachelor's or master's thesis, or any other similar document. It is styled to meet the official requirements of UiT, the Arctic University of Norway. The template is heavily influenced by [this unofficial LaTeX template.](https://github.com/egraff/uit-thesis)

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

CC: @mrtz-j

### CODE REVIEW

File: packages/preview/modern-uit-thesis/0.1.0/modules/abstract.typ [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#let abstract_page(body) = {
```
Comment (laurmaedje) [inline packages/preview/modern-uit-thesis/0.1.0/modules/abstract.typ:1]: The recommend casing convention is kebab-case, so `abstract-page`, though it's not a strict requirement.

File: packages/preview/modern-uit-thesis/0.1.0/template/chapters/global.typ [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//
+// NOTE:
+//    If you like to modify "lib.typ", copy the file from the cache directory
+//    or get it from the 'official' Typst Universe package git repo
+//       (i.e. from proper 'version number subdir' of
+//         https://github.com/typst/packages/tree/main/packages/preview/fh-joanneum-iit-thesis)
+//    Copy the lib.typ to a (sub)folder of this project and
+//    set the path accordingly.
+// #import "@preview/modern-uit-thesis:0.1.0": *
+#import "../../lib.typ": *
```
Comment (laurmaedje) [inline packages/preview/modern-uit-thesis/0.1.0/template/chapters/global.typ:23]: This results in a compiler error. (Probably also in other files.)

Comment (otytlandsvik) [reply packages/preview/modern-uit-thesis/0.1.0/template/chapters/global.typ:23]: Whoops :sweat_smile:

Comment (mrtz-j) [reply packages/preview/modern-uit-thesis/0.1.0/template/chapters/global.typ:23]: All other `typ` files that use the package import it through `global.typ`, so it should be isolated to this file.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**